Sources/Widgets/Representations/SplineContextRepresentation/index.js

```diff
@@ -65,6 +65,8 @@ function vtkSplineContextRepresentation(publicAPI, model) {
 
     const polydata = vtkPolyData.newInstance();
     const widgetState = inData[0];
+    const closed = widgetState.getSplineClosed();
+    publicAPI.setFill(closed);
```

Comment:
fill and closed are 2 independent things...

Reply:
And is it ok that when we draw a closed spline the area under the line is shown ?

Reply:
yes